theme/shadows.js

```diff
@@ -0,0 +1,80 @@
+const parseLine = line => {
+  const values = line.split(/\s+/)
+  return {
+    x: values[0],
+    y: values[1],
+    blur: values[2],
+    spread: values[3],
+    opacity: values[4]
+  }
+}
+
+const shadow1 = `0   2   4   0   8
+0   3   5   0   8
+0   4   6   0   9
+0   5   8   1   9
+0   6   9   1   9
+0   7   10  1   10
+0   8   11  1   10
+0   9   13  1   10
+0   10  14  1   11
+0   11  15  2   11
+0   12  16  2   11
+0   13  17  2   12
+0   13  19  2   12
+0   14  20  2   13
+0   15  21  2   13
+0   16  22  3   13
+0   17  23  3   14
+0   18  25  3   14
+0   19  26  3   14
+0   20  27  3   15
+0   21  28  3   15
+0   22  30  4   15
+0   23  31  4   16
+0   24  32  4   16
```

Comment:
Do we need so many variants of the shadow?

Reply:
The 24 shadows are part of the material ui specification, we want to stay as close as possible to the material ui specification. In the styleguide, I did not show so much shadows as in practice we will not use the 24 shadows.